health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CreateCampaignComponents/NewUploadData.js

```
@@ -0,0 +1,1326 @@
+  const onFileDelete = (file, index) => {
+    setUploadedFile((prev) => prev.filter((i) => i.id !== file.id));
+    setIsError(false);
+    setIsSuccess(false);
+    setIsValidation(false);
+    setApiError(null);
+    setErrorsType({});
+    setShowToast(null);
+  };
+
+  const onFileDownload = (file) => {
+    if (file && file?.url) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)